index.html

```diff
@@ -18,8 +18,16 @@
             </p>
         </div>
     </div>
-
-    <img class="site__screenshot" alt="A screenshot showing FreshRSS in action" src="images/screenshot.webp" width="1000" height="563" />
+    <!-- Slider -->
+    <div class="image-comparison-slider">
+        <div class="slider-wrapper">
+            <img class="comparison-image" alt="Light theme screenshot of FreshRSS" src="images/screenshot.webp" />
+            <img class="comparison-image overlay" alt="Dark theme screenshot of FreshRSS"
+                src="images/dark-screenshort.webp" />
+        </div>
+        <input type="range" class="slider-handle" min="0" max="100" value="50">
```

Comment:
You should add a label visually hidden to improve the accessibility of this input, something like "Select the percentage of the left image to display". You can hide the label with the class `sr-only`.